ale/drivers/mro_drivers.py

```diff
@@ -147,4 +147,16 @@ def spacecraft_name(self):
         name_lookup = {
             'MARS_RECONNAISSANCE_ORBITER': 'MRO'
         }
-        return name_lookup[self.label['SPACECRAFT_NAME']]
+        return name_lookup[super().spacecraft_name]
+
+    @property
+    def detector_start_line(self):
+        return 0
+
+    @property
+    def detector_start_sample(self):
+        return self.label.get('SAMPLE_FIRST_PIXEL', 0)
```

Comment:
I dislike this, but am not sure if we have another model for doing this? We need to value from a PDS label that is specific to this mission. I believe the decision was that the instrument/data specific driver should therefore be able to pull the data directly?

Reply:
Yes that was the decision, it can pull it directly. I see this one as, on the fence. The key from pds3 is standard, so adding a getter for it in the label_pds3 would be reasonable, but few pds3 files use it, so having the specific driver get it works also.

Reply:
Also, I'm wondering if we need to take a closer look at other docs (instrument SIS, ISIS3 code) as we create these drivers. With all the loose standards we are working with, picking a key from pds, isis, or anywhere because it sounds correct could cause problems/bugs down the line.

Reply:
All of the keys selected should be based on documentation like the SIS when we write new instrument drivers. For porting these old drivers to the new structure, we can assume that the old drivers were written correctly.

Reply:
Can someone link to the CTX SIS?

Reply:
It should be located in the top level of the PDS archive you're pulling image data from.